cmd/server/assets/codes/bulk-issue.html

```diff
@@ -111,6 +111,23 @@
         </table>
       </div>
     </div>
+    <div class="card mb-3 shadow-sm d-none" id="error-div">
+      <div class="card-header">Errors</div>
+      <div class="card-body" >
+        <h3>Errors</h3>
+        <table class="table table-bordered">
```

Comment:
Make this a full-bleed table (see API keys or users indexes for examples) and drop the h3 (it's already in the title)